examples/column/ekman.jl

```diff
@@ -0,0 +1,264 @@
+global_logger(TerminalLogger())
+
+const FT = Float64
+
+# https://github.com/CliMA/CLIMAParameters.jl/blob/master/src/Planet/planet_parameters.jl#L5
+const MSLP = 1e5 # mean sea level pressure
+const grav = 9.8 # gravitational constant
+const R_d = 287.058 # R dry (gas constant / mol mass dry air)
+const γ = 1.4 # heat capacity ratio
+const C_p = R_d * γ / (γ - 1) # heat capacity at constant pressure
+const C_v = R_d / (γ - 1) # heat capacit at constant volume
+const R_m = R_d # moist R, assumed to be dry
+
+const f = 5e-5
+const ν = 0.01
+const L = 2e2
+const nelems = 30
+const Cd = ν/(L/nelems)
+const ug = 1.0
+const vg = 0.0
+const d = sqrt(2*ν/f)
+domain = Domains.IntervalDomain(0.0, L; x3boundary = (:bottom, :top))
+#mesh = Meshes.IntervalMesh(domain, Meshes.ExponentialStretching(7.5e3); nelems = 30)
+mesh = Meshes.IntervalMesh(domain; nelems = nelems)
+
+cspace = Spaces.CenterFiniteDifferenceSpace(mesh)
+fspace = Spaces.FaceFiniteDifferenceSpace(cspace)
+
+
+function ρ_from_pθ(p, θ_liq)
+    T = θ_liq * (p / MSLP)^(R_d / C_p)
+    return p / (R_d * T)
+end
+
+# https://github.com/CliMA/Thermodynamics.jl/blob/main/src/TemperatureProfiles.jl#L115-L155
+# https://clima.github.io/Thermodynamics.jl/dev/TemperatureProfiles/#DecayingTemperatureProfile
+function adiabatic_temperature_profile(z; T_surf = 300.0, T_min_ref = 230.0)
+    ## Initial conditions
+    # Scale height for surface temperature
+
+
+
+    # Temperature
+    Γ = grav / C_p
+    T = max(T_surf - Γ * z, T_min_ref)
+
+    # Pressure
+    p = MSLP * (T / T_surf)^(grav / (R_d * Γ))
+    if T == T_min_ref
+        z_top = (T_surf - T_min_ref) / Γ
+        H_min = R_d * T_min_ref / grav
+        p *= exp(-(z - z_top) / H_min)
+    end
+        
+
+    θ = FT(T_surf)
+
+    ρ = ρ_from_pθ(p, θ)
+    ρθ = ρ * θ
+
+    u = FT(1.0)
+    v = FT(0.0)
+    return (ρ = ρ, u = u, v = v, ρθ = ρθ)
+end
+
+Π(ρθ) = C_p * (R_d * ρθ / MSLP)^(R_m / C_v)
+
+
+
+zc = Fields.coordinate_field(cspace)
+Yc = adiabatic_temperature_profile.(zc)
+w = zeros(Float64, fspace)
+
+Y_init = copy(Yc)
+w_init = copy(w)
+
+# Y = (Yc, w)
+
+function tendency!(dY, Y, _, t)
+    
+    (Yc,  w) = Y.x
+    (dYc, dw) = dY.x
+
+    UnPack.@unpack ρ, u, v, ρθ = Yc
+    dρ = dYc.ρ
+    du = dYc.u
+    dv = dYc.v
+    dρθ = dYc.ρθ
+
+    # S 4.4.1: potential temperature density
+    # Mass conservation
+
+    If = Operators.InterpolateC2F(;bottom =  Operators.Extrapolate(),
+                                   top = Operators.Extrapolate(),)
+    ∂c = Operators.GradientF2C()
+    @. dρ = -(∂c(w * If(ρ)))  # Eq. 4.11
+
+    
+    u_1 = parent(u)[1]
+    v_1 = parent(v)[1]
+    u_wind = sqrt(u_1^2 + v_1^2)
+
+    # u-momentum
+    bcs_bottom = Operators.SetValue(Cd * u_wind * u_1)  # Eq. 4.16
+    bcs_top = Operators.SetValue(FT(ug))  # Eq. 4.18
+    gradc2f = Operators.GradientC2F(top = bcs_top)
+    gradf2c = Operators.GradientF2C(bottom = bcs_bottom)
+    @. du = gradf2c(ν * gradc2f(u)) + f*(v - vg)  # Eq. 4.8
```

Comment:
if you rebase this PR on main, you should be able to use `Operators.Advection()` now

Reply:
I added the Advection operator part